docs/DeveloperGuide.md

```diff
@@ -1088,3 +1088,17 @@ Deleting an existing appointment from a patient
    Expected: An error message is shown, indicating that the command format is incorrect.
 
 [Back to Table of Contents](#table-of-contents)
+
+
+## Appendix: Planned Enhancements
+
+Team size: 5
+
+1. **Allow names to include symbols and special characters**: As of the current version of MediBase3, names are restricted to both alphanumerics and spaces only. 
+However, there are people with legal names that include symbols and special characters, such as 'Nagaratnam s/o Suppiah'. In the future, we will loosen the restrictions for names to allow users to input names with any special characters and symbols.
+
+2. **Improve `addAppt` command to allow users to add multiple appointments at once**: Currently, users can only add one appointment at a time using the `addAppt` command. We plan to enhance the `addAppt` command to allow users to add multiple appointments at once.
+This will be useful as it allows doctors to save time by adding multiple appointments for a patient in one go for patients with recurring appointments, instead of typing the same command multiple times.
```

Comment:
'patients with recurring appointments' - might be misleading / too specific. can remove